src/index.js

```diff
@@ -47,14 +45,20 @@ const parseFromStd = (stdout, stderr) => {
 const getProjectDirectory = filePath => (
   atom.project.relativizePath(filePath)[0] || path.dirname(filePath))
 
-const forwardRubocopToLinter = ({
+const getRubocopBaseCommand = command => command
+  .split(/\s+/)
+  .filter(i => i)
+  .concat(DEFAULT_ARGS)
+
+const forwardRubocopToLinter = (version, {
   message: rawMessage, location, severity, cop_name: copName,
 }, file, editor) => {
+  const latestRubocops = semver.gte(version, '0.52.0')
```

Comment:
Could you rename this to something more descriptive?

* `hasCopName`?
* `noCopNameVersion`?